modules/scylladb/src/test/java/org/testcontainers/scylladb/ScyllaDBContainerTest.java

```diff
@@ -0,0 +1,181 @@
+                .builder()
+                .addContactPoint(scylladb.getContactPoint())
+                .withLocalDatacenter("datacenter1")
+                .withSslContext(sslContext)
+                .build();
+            ResultSet resultSet = session.execute(BASIC_QUERY);
+            assertThat(resultSet.wasApplied()).isTrue();
+            assertThat(resultSet.one().getString(0)).isNotNull();
+            assertThat(session.getMetadata().getNodes().values()).hasSize(1);
+        }
+    }
+
+    @Test
+    public void testShardAwareness() {
+        try (ScyllaDBContainer scylladb = new ScyllaDBContainer(SCYLLADB_IMAGE)) {
+            scylladb.start();
+            // shardAwarenessSession {
+            CqlSession session = CqlSession
+                .builder()
+                .addContactPoint(scylladb.getShardAwareContactPoint())
+                .withLocalDatacenter("datacenter1")
+                .build();
+            // }
+            ResultSet resultSet = session.execute("SELECT driver_name FROM system.clients");
+            assertThat(resultSet.one().getString(0)).isNotNull();
+            assertThat(session.getMetadata().getNodes().values()).hasSize(1);
+        }
+    }
+
+    @Test
+    public void testAlternator() {
+        try ( // alternator {
+            ScyllaDBContainer scylladb = new ScyllaDBContainer(SCYLLADB_IMAGE).withAlternator()
+            // }
+        ) {
+            scylladb.start();
+
+            // dynamodDbClient {
+            DynamoDbClient client = DynamoDbClient
+                .builder()
+                .endpointOverride(URI.create(scylladb.getAlternatorEndpoint()))
+                .credentialsProvider(StaticCredentialsProvider.create(AwsBasicCredentials.create("test", "test")))
+                .region(Region.US_EAST_1)
+                .build();
+            // }
+            client.createTable(
+                CreateTableRequest
+                    .builder()
+                    .tableName("demo_table")
+                    .keySchema(KeySchemaElement.builder().attributeName("id").keyType(KeyType.HASH).build())
+                    .attributeDefinitions(
+                        AttributeDefinition.builder().attributeName("id").attributeType(ScalarAttributeType.S).build()
+                    )
+                    .billingMode(BillingMode.PAY_PER_REQUEST)
+                    .build()
+            );
+            assertThat(client.listTables().tableNames()).containsExactly(("demo_table"));
+        }
+    }
+
+    @Test
+    public void throwExceptionWhenAlternatorDisabled() {
+        try (ScyllaDBContainer scylladb = new ScyllaDBContainer(SCYLLADB_IMAGE)) {
+            scylladb.start();
+            assertThatThrownBy(scylladb::getAlternatorEndpoint)
+                .isInstanceOf(IllegalStateException.class)
+                .hasMessageContaining("Alternator is not enabled");
+        }
+    }
+
+    @Test
+    public void testSslConfiguration() {
```

Comment:
@mkorolyov this test fails. Lacking ssl configuration? default image doesn't provide those env vars